rebar.config.script

```diff
@@ -0,0 +1,14 @@
+HandleBigEndianPort = fun
+    F([H|T]) when element(1, H) == port_env ->
```

Comment:
Couldn't the search of `port_env` be replaced by `lists:keyfind(port_env, 1, CONFIG)`?